### PR TITLE
Explicit HTTP link for the local proxy server 

### PR DESCRIPTION
**Pull request by fqsghostcloud**
fix http link format

### CODE REVIEW

**Comment by fqsghostcloud** [comment]
@shyamjvs please merge this PR,thanks

**Comment by shyamjvs** [comment]
/lgtm
/retest

For approval, you need someone from the OWNERS file to do it.

**Comment by fqsghostcloud** [comment]
@jbeda  please merge this PR,thanks

File: pkg/kubectl/cmd/proxy.go [hunk 1]
```diff
@@ -153,7 +153,7 @@ func RunProxy(f cmdutil.Factory, out io.Writer, cmd *cobra.Command) error {
 	if err != nil {
 		glog.Fatal(err)
 	}
-	fmt.Fprintf(out, "Starting to serve on %s\n", l.Addr().String())
+	fmt.Fprintf(out, "Starting to serve on http://%s\n", l.Addr().String())
```
Comment (soltysh) [inline pkg/kubectl/cmd/proxy.go]: This assumes it's always http, but that's not a good assumption.

**Review by soltysh — CHANGES_REQUESTED** [review]
I don't think this kind of change is necessary.

**Pull request closed without merging**